src/Http/Controllers/TwoFactorAuthenticatedSessionController.php

```diff
@@ -61,6 +61,11 @@ public function store(TwoFactorLoginRequest $request)
 
             event(new RecoveryCodeReplaced($user, $code));
         } elseif (! $request->hasValidCode()) {
+            $request->session()->put([
+                'login.id' => $user->getKey(),
+                'login.remember' => $request->filled('remember'),
```

Comment:
Why not `$request->remember()`?

Reply:
@taylorotwell you're right, at this point ```$request->filled('remember')``` not going to help. It needs to be ```$request->remember()```.